recipes/CAMPneu/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "CAMPneu" %}
+{% set version = "1.1.0" %}
+{% set hash = "1f2bc9dbb9fa71fce9876852ba0dedad997cca800734b81a6a43fff8e5e20ac3" %}
+{% set user = "appliedbinf" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ user }}/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: {{ hash }}
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - skip: True  # [win]
```

Comment:
This should be part of `root > build`, not `root > requirements > build`

Reply:
Please do not resolve conversations if you haven't addressed the issue.